assets/controllers/planning-scroll_controller.js

```diff
@@ -0,0 +1,32 @@
+import { Controller } from "@hotwired/stimulus";
+
+/** Scroll to active sprint in planning view */
+export default class extends Controller {
+    static targets = ["column"];
+
+    connect() {
+        const activeColumn = this.columnTargets.find((el) => el.dataset.active);
+
+        if (!activeColumn) {
+            return;
+        }
+
+        const activeIndex = activeColumn.dataset.index;
+        const scrollToIndex = Math.max(0, activeIndex - 1);
+        const scrollToColumn = this.columnTargets.find(
+            (el) => el.dataset.index == scrollToIndex,
+        );
+        const firstColumn = this.columnTargets.find(
+            (el) => el.dataset.index == 1,
+        );
+
+        if (!scrollToColumn || !firstColumn) {
+            return;
+        }
+
+        const { x } = scrollToColumn.getBoundingClientRect();
+        const { x: firstColumnX } = firstColumn.getBoundingClientRect();
```

Comment:
It took me a while to actually understand what's going on here.

```suggestion
        const x = scrollToColumn.getBoundingClientRect().x; 
        const firstColumnX = firstColumn.getBoundingClientRect().x;
```

is much easier to understand.

Reply:
Changed it. Thanks for the suggestion.